### DISCUSSION TITLE
ToLiss A319 Profile

### DISCUSSION THREAD

**Discussion by Aviodoc** [mauer/xmidictrl#11] [Community Aircraft Profiles]
Hello! I love your plug-in and I am using it very much with most of the aircraft already supported. 
I am wondering if there any chance you could make a ToLiss A319 profile or there is a simple way for me to adapt the A321 profile to the A319.
Thank you very much!

**Comment by mauer** [comment]
Hi Nefass,

Many thanks for the kind words! I don't own the A319, but I know that both aircrafts are very similar are they use the same systems engine. So chances are high, that the profile will work fine. I just had a look at all the mappings and none of them is labelled with A321.

Just take the A321_XMidiCtrl.toml file, rename it to XMidiCtrl.toml and place it into the A319 aircraft folder. Let me know if there are any problems. Would be cool indeed, to have support for both aircrafts :-)

Thanks,
Marco

**Reply by Aviodoc** [reply]
Hi Marco, thank you for your fast answer. 
I tried what you wrote me and indeed...it worked!! 
Thank you for your support. I did not find a printable layout for the 321, could you please include it in the next release?
